### ISSUE TITLE
How do I fix the bottom layout on Android?

### ISSUE THREAD

**Issue by DenMarty** [nick92/flutter_mapbox#4]
First I want to thank you for this plugin, it helped me a lot.

The only problem is the display of information at the bottom of the screen on Android (Info Panel).
How can I fix the indents and align the elements? Or do you plan to fix this in future versions?

Thanks!

![Screenshot](https://user-images.githubusercontent.com/32595813/198334319-7dd597cd-3834-472c-9c01-b2729d0a472f.jpg)

**Comment by nick92** [comment]
How would you like this view to appear? it's something that would need amending in the map_activtiy.xml file within the Android project

**Comment by DenMarty** [comment]
Thanks, I already found it and made some changes. I just wanted this block to look neater on the users devices. The only problem is that all changes are overwritten when the package is updated.

**Comment by nick92** [comment]
Ok, send me over the changes made and a screenshot and I’ll look to incorporate them in the next release

**Comment by DenMarty** [comment]
I just made a few minor changes to map_activity to correct the indentation and align the text.
I think that would be enough.

```
<?xml version="1.0" encoding="utf-8"?>
<androidx.constraintlayout.widget.ConstraintLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="match_parent"
    android:layout_height="match_parent">

    <com.mapbox.maps.MapView
        android:id="@+id/mapView"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toTopOf="parent" />


    <androidx.cardview.widget.CardView
        android:id="@+id/tripProgressCard"
        android:layout_width="0dp"
        android:layout_height="wrap_content"
        android:layout_margin="0dp"
        app:cardElevation="0dp"
        app:cardUseCompatPadding="false"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintStart_toStartOf="parent">

        <com.mapbox.navigation.ui.tripprogress.view.MapboxTripProgressView
            android:id="@+id/tripProgressView"
            android:layout_width="match_parent"
            android:paddingTop="15dp"
            android:paddingBottom="15dp"
            android:paddingStart="20dp"
            android:paddingEnd="20dp"
            android:layout_height="wrap_content"
            android:layout_gravity="end|center_vertical" />

        <ImageView
            android:id="@+id/stop"
            android:layout_width="30dp"
            android:layout_height="30dp"
            android:layout_gravity="end|center_vertical"
            android:layout_marginEnd="12dp"
            app:srcCompat="@android:drawable/ic_menu_close_clear_cancel" />

    </androidx.cardview.widget.CardView>

    <com.mapbox.navigation.ui.maneuver.view.MapboxManeuverView
        android:id="@+id/maneuverView"
        android:layout_width="0dp"
        android:layout_height="wrap_content"
        android:layout_margin="0dp"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toTopOf="parent" />

    <com.mapbox.navigation.ui.voice.view.MapboxSoundButton
        android:id="@+id/soundButton"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_marginTop="8dp"
        android:layout_marginEnd="16dp"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintTop_toBottomOf="@id/maneuverView" />

    <com.mapbox.navigation.ui.maps.camera.view.MapboxRouteOverviewButton
        android:id="@+id/routeOverview"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_marginTop="8dp"
        android:layout_marginEnd="16dp"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintTop_toBottomOf="@id/soundButton" />

    <com.mapbox.navigation.ui.maps.camera.view.MapboxRecenterButton
        android:id="@+id/recenter"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_marginTop="8dp"
        android:layout_marginEnd="16dp"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintTop_toBottomOf="@id/routeOverview" />


</androidx.constraintlayout.widget.ConstraintLayout>
```

**Comment by nick92** [comment]
Ok looks good, will roll this into the next release

**Comment by nick92** [comment]
That view change is included from version 0.3.7 and on
